ccd-definition/CaseTypeTab.json

```diff
@@ -26,9 +26,19 @@
     "TabID": "CasePeopleTab",
     "TabLabel": "People in the case",
     "TabDisplayOrder": 2,
-    "CaseFieldID": "children",
+    "CaseFieldID": "respondentsMigrated",
```

Comment:
Do we need to have it here? I think end user is not interested in knowing whether migration is complete or not.

Reply:
Let me test - I think this may be left over from me trying to get data to appear in the tabs. But unsure

Reply:
can be removed 👍 good spot

Reply:
Keeping in to add temporary show hide to old respondents after migration